### ISSUE TITLE
Maximum number of windows limited to 100

### ISSUE THREAD

**Issue by brandy125** [gnudatalanguage/gdl#1619]
Any chance to allow more than 100 windows in GDL?

In GDL we are limited to 69 "free" windows:

```
GDL> for i=0,200 do window,/free
% WINDOW: No more window handles left.
% Execution halted at: $MAIN$          
GDL> help,i
I               INT       =       69
GDL> 
```

In IDL there does not seem to exist this limit:

```
IDL> for i=0,200 do window,/free
IDL> help,i
I               INT       =      201
IDL> 
```

**Comment by GillesDuvert** [comment]
unfortunately this is a (silly) hard plplot limit, plplot allows 101 windows. Apparently.
One could patch plplot (the number is defined in a .h) and report the new number in GraphicsMultiDevice::WAddFree(), line 500 of graphicsdevice.cpp
Upon success, adding a patch to plplot is not impossible.

**Comment by GillesDuvert** [comment]
closed by #1886
